templates/components/common/navigation.html

```diff
@@ -18,6 +18,14 @@
         <li class="navUser-item">
             <a class="navUser-action navUser-item--compare" href="{{urls.compare}}" data-compare-nav>{{lang 'common.compare'}} <span class="countPill countPill--positive countPill--alt"></span></a>
         </li>
+        {{#if theme_settings.social_icon_placement_top}}
+             <li class="navUser-item">   
+             <span class="navUser-item-social"> 
+                {{> components/common/social-links}} 
+             </span>   
+            </li>
+             <li class="navUser-item"> <span class="navUser-item-social-break">    |</span></li>
```

Comment:
You should be able to solve this with CSS only. Is there a reason we're using new markup?